package.json

```diff
@@ -17,7 +17,9 @@
         "build-local:frontend": "enyo pack",
         "build-local:service": "webpack",
         "build-local:package": "ares-package dist/ -e enyo-ilib",
-        "build-and-launch": "npm-run-all --serial build-local deploy launch"
+        "build-and-launch": "npm-run-all --serial build-local deploy launch",
+        "version": "node tools/sync-version.js && git add appinfo.json",
+        "postversion": "git push --tags"
```

Comment:
Do we need automatic push? I prefer to leave this step to be done manually.

Reply:
Not really, I can remove that.